src/controller/CommissioneeDeviceProxy.h

```diff
@@ -237,6 +237,7 @@ class CommissioneeDeviceProxy : public DeviceProxy, public SessionReleaseDelegat
     }
 
     ByteSpan GetCSRNonce() const { return ByteSpan(mCSRNonce, sizeof(mCSRNonce)); }
+    MutableByteSpan GetCSRNonce() { return MutableByteSpan(mCSRNonce, sizeof(mCSRNonce)); }
```

Comment:
Why this overload? GetCSRNonce() is to have the storage. This should not be mutated here through a MutableByteSpan accessor.

Reply:
Addressed in latest commit